src/hooks/useValidation.js

```diff
@@ -0,0 +1,69 @@
+export default function useValidation() {
+  const validationFunc = (name, value) => {
+    switch (name) {
+      case 'name':
+        return nameValidation(value);
+      case 'description':
+        return descriptionValidation(value);
+      case 'price':
+        return priceValidation(value);
+      case 'tag':
+        return tagValidation(value);
+      default:
+        return 'Invalid input name';
+    }
+  };
+
+  return validationFunc;
+}
+
+function nameValidation(value) {
+  let errMsg;
+  if (value?.length <= 0) {
```

Comment:
@csbizz length는 음수가 될 수 없고, string 이라면 '' 도 0과마찬가지로 falsy 이므로 아래처럼 작성할 수 있을거같아요
```suggestion
  if (!value?.trim?.()) {
```